k8s/prod/common/ccd/definition-store-api.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/definition-store-api:prod-77f5956b
       environment:
-        IDAM_USER_URL: "https://hmcts-access.service.gov.uk"
+       ELASTIC_SEARCH_FAIL_ON_IMPORT: false
```

Comment:
Please keep IDAM_USER_URL.. you need that

Reply:
done